src/mo_pack/tests/test_rle.py

```diff
@@ -6,6 +6,7 @@
 
 import numpy as np
 from numpy.testing import assert_array_equal
+from numpy.typing import NDArray
```

Comment:
@ukmo-ccbunney You should be able to adopt the pattern:

```python
from typing import TYPE_CHECKING

if TYPE_CHECKING:
    from numpy.typing import NDArray
```

Reply:
Cool - done. Took me a few mins to realise I needed to protect the types from the runtime during normal execution by putting them in quotes.